app/views/tripfamilies/_people_to_allocate.html.erb

```diff
@@ -0,0 +1,7 @@
+<tr>
+  <td><%= @tripfamily.to_allocate(params[:trip_id])["girls"]%></td>
+  <td><%= @tripfamily.to_allocate(params[:trip_id])["boys"]%></td>
+  <td><%= @tripfamily.to_allocate(params[:trip_id])["teachers"]%></td>
+  <td><%= @tripfamily.to_allocate(params[:trip_id])["bus_drivers"]%></td>
+  <td><%= @trip.school_name %></td>
+</tr>
```

Comment:
I'm guessing you may be yet to do translations - not sure you need them as Im not sure how the above displays, e.g. ["girls"]

Reply:
I'm sure it looks confusing, but it display only numbers 😉